packages/venia-ui/lib/components/ProductOptions/option.js

```diff
@@ -80,7 +78,7 @@ Option.propTypes = {
     }),
     label: string.isRequired,
     onSelectionChange: func,
-    selectedVariant: object,
+    selectedValue: oneOfType([number, string]), // TODO: can these be objects?
```

Comment:
@supernova-at off the cuff do you recall whether these values are always
```js
{
  value: <string or number>
  label: <string>
}
```

Reply:
That looks correct to me, yeah

Reply:
```suggestion
    selectedValue: oneOfType([number, string]),
```